perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -24,6 +25,33 @@
 
 public class AuthzResolver {
 
+	/**
+	 * Check if the principal is authorized.
+	 *
+	 * @param sess perunSession which contains the principal.
+	 * @param policyDefinition of policy which contains authorization rules.
+	 * @param objects on which will be authorization provided.
+	 * @return true if the principal has particular rights, false otherwise.
+	 * @throws PolicyNotExistsException when the given policyDefinition does not exist in the PerunPoliciesContainer.
+	 */
+	public static boolean authorized(PerunSession sess, String policyDefinition, PerunBean... objects) throws PolicyNotExistsException {
+		return AuthzResolverBlImpl.authorized(sess, policyDefinition, objects);
+	}
+
+	/**
+	 * Check if the principal is authorized.
+	 *
+	 * @param sess perunSession which contains the principal.
+	 * @param policyDefinition of policy which contains authorization rules.
+	 * @param listOfObjects on which will be authorization provided.
+	 * @param objects on which will be authorization provided.
+	 * @return true if the principal has particular rights, false otherwise.
+	 * @throws PolicyNotExistsException when the given policyDefinition does not exist in the PerunPoliciesContainer.
+	 */
+	public static boolean authorized(PerunSession sess, String policyDefinition, List<? extends PerunBean> listOfObjects, PerunBean... objects) throws PolicyNotExistsException {
+		return AuthzResolverBlImpl.authorized(sess, policyDefinition, listOfObjects, objects);
```

Comment:
There must be more information in javadoc. From definition of:
```
@param listOfObjects on which will be authorization provided.
@param objects on which will be authorization provided
```
I am not able to say, what should be put there to check specific situation. Better description of parameters with examples could really help here.

Reply:
Method authorized() now takes only List<PerunBean>.